src/cml.js

```diff
@@ -69,8 +70,41 @@ const getDriver = (opts) => {
   throw new Error(`driver ${driver} unknown!`);
 };
 
+const fixGitSafeDirectory = () => {
+  const getOrSetGitConfigSafeDirectory = (value) =>
+    spawnSync(
+      'git',
+      [
+        'config',
+        '--global',
+        value ? '--add' : '--get-all',
+        'safe.directory',
+        value
+      ],
+      {
+        encoding: 'utf8'
+      }
+    ).stdout.split(/[\r\n]+/);
+
+  const addSafeDirectoryIdempotent = (directory) =>
+    getOrSetGitConfigSafeDirectory().includes(directory) ||
+    getOrSetGitConfigSafeDirectory(directory);
+
+  addSafeDirectoryIdempotent('/');
+  addSafeDirectoryIdempotent('*');
+  for (
+    let root, dir = process.cwd();
+    root !== dir;
+    { root, dir } = path.parse(dir)
+  ) {
+    addSafeDirectoryIdempotent(dir);
+  }
+};
+
 class CML {
   constructor(opts = {}) {
+    fixGitSafeDirectory();
```

Comment:
put a link to the ticket about as a comment?

Reply:
```suggestion
    fixGitSafeDirectory(); // https://github.com/iterative/cml/issues/970
```